packages/itwin/tree-widget/src/components/tree-header/TreeHeader.scss

```diff
@@ -66,4 +70,38 @@
       width: var(--iui-size-xl);
     }
   }
+
+  &.enlarge {
+    --enlarged-header-height: calc(var(--iui-size-l) + var(--iui-size-m) + var(--iui-size-3xs));
+
+    height: var(--enlarged-header-height);
+    min-width: calc(100% - var(--enlarged-header-height));
+
+    .tree-widget-search-box {
+      height: var(--enlarged-header-height);
+      width: calc(100% - var(--enlarged-header-height));
+
+      &.contracted {
+        width: var(--enlarged-header-height);
+      }
+
+      .search-open, .search-input, .search-close {
+        height: var(--enlarged-header-height);
+        min-width: var(--enlarged-header-height);
+      }
+    }
+
+    .button-container {
+      width: calc(100% - var(--enlarged-header-height));
+
+      .header-button {
```

Comment:
Can we adjust header to accommodate button component from `itwinui-react` with default size?

Reply:
Switched to using larger buttons, so we would no longer need to additionally increase the size.